### PR TITLE
Support restore wallet from seed

### PR DESCRIPTION
**Pull request by gdmmx**
### Proposed changes in this pull request
New feature for restore wallet from seed via nknc

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [x] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [x] Read the [CONTRIBUTION guidelines](https://github.com/nknorg/nkn#contributing).
- [ ] All the tests are passing after the introduction of new changes.
- [x] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable (in code and README.md).
- [x] Code has been written according to [NKN-Golang-Style-Guide](https://github.com/nknorg/nkn/wiki/NKN-Golang-Style-Guide)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

File: cmd/nknc/wallet/wallet.go [hunk 1]
```diff
@@ -79,29 +78,57 @@ func walletAction(c *cli.Context) error {
...
+		break
+	case "hex":
+		hexFmt = true
+		fallthrough
+	case "bin":
+		if common.FileExisted(name) {
+			return fmt.Errorf("CAUTION: '%s' already exists!\n", name)
+		}
+
+		key, err := password.GetPassword("Input you PrivateKey")
```
Comment (yilunzhang) [inline cmd/nknc/wallet/wallet.go:103]: Should be seed instead of PrivateKey

Comment (gdmmx) [reply cmd/nknc/wallet/wallet.go:103]: I worry about that user will confuse with crypto seed(nknc) & DHT node seed(nknd).

Comment (yilunzhang) [reply cmd/nknc/wallet/wallet.go:103]: Then we should use `Secret Seed`. Private key is different from seed in Ed25519

Comment (gdmmx) [reply cmd/nknc/wallet/wallet.go]: Done.

**Pull request merged**